server/validators/prePostAppointmentValidator.ts

```diff
@@ -28,27 +27,15 @@ export const PrePostAppointmentValidator: Validator = (body: Record<string, stri
     errors.push({ text: 'Select a room for the post-court hearing briefing', href: '#postAppointmentLocation' })
   }
 
-  if (!bvlsMasteredVlpmFeatureToggleEnabled() && !body.bookingType) {
-    errors.push({ text: 'Select a booking type', href: '#bookingType' })
-  }
-
-  if ((bvlsMasteredVlpmFeatureToggleEnabled() || body.bookingType === 'COURT') && !body.court) {
+  if (!body.court) {
```

Comment:
I don't see the check for `body.bookingType == 'COURT'` in the above check anymore? It was a switch OR, so maybe always triggered now, hence removed. Probably right.